README.md

```diff
@@ -111,6 +111,33 @@ Given input like:
 * | json | logfmt from nested_key | fields some
 ```
 
+##### Split
+`split [input_field] [with separator] [as new_field]`: Split the input via the separator (default is `,`). Output is an array type. If no `input_field` or `new_field`, the contents will be put in the key `_split`.
+
+*Examples*:
+```agrind
+* | split with " "
```

Comment:
lets do `split on ...` instead of `split with`

Reply:
👍 Done!